### ISSUE TITLE
Allow swap into other crypto

### ISSUE THREAD

**Issue by Benskalz** [Nault/Nault#289]
Services like [nano.trade](https://nano.trade/) make it possible to buy Banano or Moon coin very easily, almost instantly and doesn't need any registration. Trade with Nano fork coins make it possible to exchange Nano into other coins in only few seconds by trusting only one provider, it offers a really good user experience. 
It could be very great to be able to swap Nano into other crypto directly from wallet by choosing wich providers we want to use.

**Comment by Joohansson** [comment]
This will most likely not be integrated into Nault since it's a decentralized wallet in that sense there is no direct support or responsible entity to handle such thing.

**Comment by BitDesert** [comment]
A simple Buy/Sell widget would be possible with a third party service like MoonPay/Changelly etc just like WeNano did.
But providers like nano.trade do not offer the APIs and reliability needed for such a feature.

**Comment by Benskalz** [comment]
> This will most likely not be integrated into Nault since it's a decentralized wallet in that sense there is no direct support or responsible entity to handle such thing.

Yes I understand, maybe with a warning message to say that this is a third party and it's not Nault responsabilities, and let user choose wich provider he wants to trust/use ?


> A simple Buy/Sell widget would be possible with a third party service like MoonPay/Changelly etc just like WeNano did.
> But providers like nano.trade do not offer the APIs and reliability needed for such a feature.

I asked to nano.trade dev and he said me that he is working to release a public API. 
I like nano.trade because it's simply the fastest exchange that I know to exchange 2 differents crypto wallet to wallet, and the only exchange who show the real potential of Nano, but yes I do understant that this may be in too early stage to be integrated right now in Nault wallet.
